target/linux/x86/image/Makefile

```diff
@@ -217,14 +325,22 @@ define Image/Build
 	$(call Image/Build/$(1))
   ifneq ($(1),iso)
 	$(call Image/Build/grub2,$(1))
+	$(call Image/Build/efi,$(1))
```

Comment:
This doesn't depend on ``ifeq ($(1),ext4)``. This means the UEFI images will be generated for both ext4 and squashfs image.

Does the generated squashfs image boot on EFI system?

I put the conditional ext4 so that the UEFI image is generated only for ext4 filesystem since I can't get the squashfs image to boot on EFI system.